akka-actor/src/main/scala/akka/dispatch/AbstractDispatcher.scala

```diff
@@ -365,6 +363,19 @@ abstract class MessageDispatcherConfigurator(_config: Config, val prerequisites:
         new ThreadPoolExecutorConfigurator(config.getConfig("thread-pool-executor"), prerequisites)
       case "affinity-pool-executor" =>
         new AffinityPoolConfigurator(config.getConfig("affinity-pool-executor"), prerequisites)
+      case "virtual-thread-executor" =>
+        val executorConfig = config.getConfig("virtual-thread-executor")
+        if (VirtualThreadConfigurator.virtualThreadsSupported()) {
+          new VirtualThreadConfigurator(executorConfig, prerequisites)
+        } else {
+          val fallbackExecutorName = executorConfig.getString("fallback")
+          if (fallbackExecutorName.isEmpty)
+            throw new RuntimeException(
+              s"Dispatcher configured to use virtual threads, but JVM version ${JavaVersion.majorVersion} does not support that. " +
+              "Use a newer Java version (21 or later), or configure 'fallback' for alternative executor on older Java versions.")
```

Comment:
Where should I configure the fallback?
Maybe better to add the full path to this settings.

Reply:
We don't know the full path here, we have the `Config` for the dispatcher, but that does not include the dispatcher name or full path to it, so I think this together with documentation in `reference.conf` is the best we can do.

Reply:
We could say `virtual-thread-executor.fallback` though, I'll add that

Reply:
I see. I was indeed thinking about the fullpath, but the config could be at any level. 
A partial path will already help.